### DISCUSSION TITLE
How to cleanup CephFS subvolumes?

### DISCUSSION THREAD

**Discussion by uhthomas** [rook/rook#12003] [General]
I had `preserveFilesystemOnDelete: true` configured on a `CephFilesystem`, which is fine. I deleted the PVCs and PVs, and the directories in the `csi` subvolume still exist, which is fine. I've confirmed I definitely don't want them now, so how can I delete those directories? I've looked around and haven't been able to find much help.

Appreciate your time! Thanks.

**Comment by sp98** [comment]
check if [this](https://github.com/ceph/ceph-csi/blob/devel/docs/resource-cleanup.md) helps.

**Reply by uhthomas** [reply]
That does help, thank you.

```sh
bash-4.4$ ceph fs subvolume rm mainfs-ec csi-vol-00cb9e0c-4dff-49cb-813e-0d12ee93400b csi
```
